packages/ui/src/proposals/constants/proposals.ts

```diff
@@ -0,0 +1,9 @@
+export const ProposalStages = {
+  deciding: { title: 'Deciding', desc: 'Lorem ipsum, dolor sit amet consectetur' },
+  dormant: { title: 'Dormant', desc: 'Lorem ipsum, dolor sit amet consectetur' },
+  gracing: { title: 'Gracing', desc: 'Lorem ipsum, dolor sit amet consectetur' },
+  succeded: { title: 'Execution Succeded', desc: 'Lorem ipsum, dolor sit amet consectetur' },
+}
+export const ProposalTypes = {
+  funding: { title: 'Funding Request', desc: 'Lorem ipsum, dolor sit amet consectetur' },
+}
```

Comment:
We can keep it for now but in the `Proposal` format:

```
export const proposals: Proposal[] = [
   { id: 0, stage: 'DECIDING', type: 'Founding request', title: 'Lorem ipsum foo bar', proposer: { id: '0', ... } },
   { id: 1, stage: 'DORMANT', type: 'Founding request', title: 'Lorem ipsum foo bar', proposer: { id: '0', ... } },
]
```

For the helper text, that can be also the same for all and inlined.

Reply:
This can be also reused in a hook later on: [https://github.com/Joystream/pioneer/issues/541#issuecomment-841236089](https://github.com/Joystream/pioneer/issues/541#issuecomment-841236089).